Qualitätsanforderungen.md

```diff
@@ -0,0 +1,13 @@
+## Qualitätsanforderungen
+
+* Kommentare und Variablendeklarationen werden in Englisch verfasst.
```

Comment:
Variablen auf jeden Fall, bei Kommentaren bin ich mir nicht so sicher. Aus den (javadoc-)Kommentaren wird ja auch die Dokumentation erstellt, und ich weiß nicht in welcher Sprache die sein soll

Reply:
Ich habe es jetzt mal so allgemein angepasst, dass wir später in der Lage sind, die javadoc-Kommentare in beliebiger Sprache in eine Dokumentation zu schreiben.